scrapers/freshmaker.py

```diff
@@ -87,7 +78,15 @@ def query_api_and_update_neo4j(self):
 
                 event.conditional_connect(event.triggered_by_advisory, advisory)
 
-                for build_dict in fm_event['builds']:
+                event_builds_url = ('https://freshmaker.engineering.redhat.com/api/2/builds/'
```

Comment:
How about the following approach? It's a little awkward since Python doesn't have a do while loop though.
```python
event_builds_url = 'https://freshmaker.engineering.redhat.com/api/2/builds/?event_id={0}&per_page=50'.format(fm_event['id'])
builds = []
while True:
    rv = session.get(event_builds_url, timeout=60).json()
    builds.extend(rv['items'])
    if rv['meta']['next']:
        event_builds_url = rv['meta']['next']
    else:
        break
```

There's nothing technically wrong with your approach, but I find it unconventional that you are altering the JSON returned by the Freshmaker API. In addition, the duplicate the `session.get` method call with your approach is unnecessary in this case. Feel free to change the above example to your style though (e.g. changing variable names).